static-build/cmake/Package.cmake

```diff
@@ -0,0 +1,106 @@
+# CPack helps to create packages for deb and rpm based distros. 
+# Creating rpm packages depends on package rpm-build. Deb package doesn't
+# need any additional packages. CPack should be version 3.1 or greater.
+# For building packages, you should use centos 7 for save compatibility.
+# cmake3 and cpack3 have bug in centos 7 which add build artifacts to package.
+# To avoid this bug we create 3 packages: tarantool, tarantool-devel and
+# tarantool-slim. The first package contains only binary files and building artifacts. 
+# Second package contains only header files. Third package contains only binary
+# file tarantool.
+
+message(VERSION " $ENV{VERSION}")
+
+execute_process(COMMAND uname -m OUTPUT_VARIABLE ARCHITECTURE)
```

Comment:
You can use the `CMAKE_SYSTEM_PROCESSOR` variable instead. It is a built-in variable.